kitsune/users/models.py

```diff
@@ -115,7 +115,6 @@ class Meta(object):
         permissions = (
             ("view_karma_points", "Can view karma points"),
             ("deactivate_users", "Can deactivate users"),
-            ("screen_share", "Can screen share"),
```

Comment:
You need to run `./manage.py makemigrations` given this change, and commit the resulting file.

Reply:
I've committed migration file. Can you take a look into pull request, once again?